codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionDecoder.java

```diff
@@ -49,56 +49,38 @@
     /**
      * Builder for instances of {@link DefaultHttp2ConnectionDecoder}.
      */
-    public static class Builder implements Http2ConnectionDecoder.Builder {
+    public static class Builder {
```

Comment:
@nmittler could we just get rid of the builder altogether? It's only 5 arguments and we are not doing anything special in the builder. Why not pass those arguments to the constructor directly?